angular/src/app/screens/login/login.component.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { CommonModule } from '@angular/common';
```

Comment:
You should check what happened when there is a missing field and when credentials are wrong 💯